### PR TITLE
Create LICENSE.mg

### PR DESCRIPTION
**Pull request by anenriquez**
Create LICENSE.mg

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+  If you develop a new program, and you want it to be of the greatest
+possible use to the public, the best way to achieve this is to make it
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+state the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    fleet_management
```
Comment (argenos) [inline LICENSE.md:634]: Update this to fit your repo! You can also do it through [GitHub](https://docs.github.com/en/github/building-a-strong-community/adding-a-license-to-a-repository)'s interface (and commit directly to master).

**Review by argenos — CHANGES_REQUESTED** [review]
I think it's better if you add the file through GitHub's

File: LICENSE.md (hunk 1 above)
Comment (anenriquez) [reply LICENSE.md:634]: Thanks! I am trying to commit directly with the GitHub's interface, but master is protected. I can open a new MR with the license created by the GitHub's interface.

Comment (argenos) [reply LICENSE.md:634]: Yeah, that sounds good

**Pull request closed without merging**